CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added decoding invitiations from a valid URL format
+- Added return routing support for connection establishment
+- Updated AgentFramework to 4.0.0-preview
```

Comment:
In `### Updated` section and include the preview build no?

Reply:
Because it is unreleased perhaps it is better to omit this from the changelog until we have a version for an update to occur from?

Reply:
Yeah, agreed